resources/todomvc/vanilla-examples/javascript-es5/src/store.js

```diff
@@ -43,20 +43,20 @@
      * });
      */
     Store.prototype.find = function (query, callback) {
-        if (!callback) {
+        if (!callback)
             return;
-        }
 
         var todos = JSON.parse(MemoryStorage[this._dbName]).todos;
 
         callback.call(
             this,
             todos.filter(function (todo) {
                 for (var q in query) {
-                    if (query[q] !== todo[q]) {
+                    if (query[q] !== todo[q])
                         return false;
-                    }
                 }
+
+
```

Comment:
This looks like an extra blank line?

Reply:
removed - ran format on root again and the extra space didn't re-appear. 
must have been a manual clean-up that introduced it.